src/main/java/seedu/duck/task/TaskType.java

```diff
@@ -0,0 +1,7 @@
+package seedu.duck.task;
+
+public enum TaskType {
```

Comment:
Oh I just realized you actually did declare an enumeration for this. You can use this in your `switch-case` statement above.